telco-hub/LSO/namespace.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hi @mmorency2021 can you add correct content for this CR. I know there are additional PRs following, but each should contain a set of valid content. 
Thanks!